internal/compiler-interface/src/main/contraband/incremental.json

```diff
@@ -103,13 +103,6 @@
           ]
         },
         {
-          "name": "nameHashing",
```

Comment:
> I think that to make migration easier we should describe all the breaking changes in a document.

Migration from what to what?

In https://github.com/sbt/sbt/pull/2969 I'm trying to deal with breaking changes between zinc 1.0.0-X8 and zinc 1.0.0-X9 and suffering the consequences. Are you going to write me a migration guide for that?

> But I don't think we have to use deprecations, we're doing a face lift to Zinc so this is the right moment to remove deprecations and do breaking changes. Otherwise we're doomed to maintain the previous, unstable API.

You're free to define your deprecation cycles to have the version or time span you see fit. Just tell me what they are. If it's as quick as "deprecated in X9, removed in X10, so you might have to upgrade to X9 before upgrading to X10" that's fine. At least I don't have to reverse engineer Zinc pull requests to get sbt to compile.